### ISSUE TITLE
Do not use own copy of actionProxy.py use common one from dockerskeleton

### ISSUE THREAD

**Issue by csantanapr** [apache/incubator-openwhisk-runtime-swift#15]
We currently have our own copy of [actionProxy/actionproxy.py](https://github.com/apache/incubator-openwhisk-runtime-swift/blob/master/core/actionProxy/actionproxy.py#L50-L54)
https://github.com/apache/incubator-openwhisk-runtime-swift/blob/master/core/actionProxy/actionproxy.py#L186

We should instead use the one from dockerSkeleton, but for swift4 we need to update dockerSkeleton to take a fourth parameter `zipdest` to be able to override the destination of the zip when unzipping the action.

This snippet needs to be push upstream to runtime docker and then pull down from Dockerfile
```python
    def __init__(self, source=None, binary=None, zipdest=None):
        defaultBinary = '/action/exec'
        self.source = source if source else defaultBinary
        self.binary = binary if binary else defaultBinary
        self.zipdest = zipdest if zipdest else os.path.dirname(self.source)
```

Once that's availabl we can update Dockerfile with
```Dockerfile
# Add the action proxy
RUN mkdir -p /actionProxy
ADD https://raw.githubusercontent.com/apache/incubator-openwhisk-runtime-docker/dockerskeleton%401.0.0/core/actionProxy/actionproxy.py /actionProxy/actionproxy.py
```

**Comment by csantanapr** [comment]
Depends on https://github.com/apache/incubator-openwhisk-runtime-docker/issues/13

**Comment by csantanapr** [comment]
Closed by #13
